test/integration/targets/collections/runme.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eux
 
-export ANSIBLE_INSTALLED_CONTENT_ROOTS=$PWD/collection_root_user,$PWD/collection_root_sys
+export ANSIBLE_COLLECTION_PATHS=$PWD/collection_root_user,$PWD/collection_root_sys
```

Comment:
Why not colon? `:`

Reply:
pathspec type already handles : notation and returns a list, since ',' is allowed in file names we try to avoid using lists

Reply:
Sorry, I don't get it. What is the purpose of `,` vs. `:` then?

Reply:
@nitzmahone can we please have a `pathspec` parser here instead?

Reply:
done